src/en_us/index.js

```diff
@@ -27,12 +29,16 @@ export const i19alt = 'Alt tag'
 export const i19alterationPrice = 'Default price alteration on products with this grid for customization or variations.'
 export const i19amount = 'Amount'
 export const i19another = 'Another'
+export const i19api = 'Api'
```

Comment:
Se escreve com tudo maiúsculo...

```suggestion
export const i19api = 'API'
```